Apps/Src/IOState.c

```diff
@@ -0,0 +1,92 @@
+#include "common.h"
+#include "os_cfg_app.h"
+#include "IOState.h"
+#include "Tasks.h"
+#include "Pedals.h"
+#include "Ignition.h"
+#include "Dashboard.h"
+#include "StatusLeds.h"
+
+static void putIOState(void);
+
+#define IO_STATE_DLY_MS 250u 
+#define IO_STATE_HEARTBEAT_DELAY (1000u)/IO_STATE_DLY_MS
+
+void putIOState(void){
+    CANDATA_t message;
+
+    // Clear message before sending
+    memset(&message, 0, sizeof message);
```

Comment:
```suggestion
    memset(&message, 0, sizeof message);
```
Consider not using a memset for a stack variable. It incurs an uneccessary cost. Just do a null init like:
```
CANDATA_t message = {
  .ID = IO_STATE,
  .idx = 0,
  .data = {0}
};
```
```

Reply:
I like having an explicit line to clear it all to be 0, easier to follow logic and reasonings

Reply:
nah